packages/meditrak-app/app/assessment/specificQuestions/GeolocateQuestion.js

```diff
@@ -57,7 +57,10 @@ class GeolocateQuestionComponent extends React.Component {
     if (isFindingLocation) {
       this.updateAnswer();
 
-      if (userLocation.errorMessage) {
+      if (
+        userLocation.errorMessage &&
+        this.props.answer.errorMessage !== userLocation.errorMessage
```

Comment:
This is for when the retry button has just been clicked, so that the error isn't immediately set again before "refreshWatchingUserLocation" has a chance to take effect.